components/BankPayouts/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import Image from 'next/image'
+import HandWithPenny from 'public/images/handWithPenny.png'
+import LzButton from 'components/UI/button'
+import Styles from './index.module.scss'
+import classNames from 'classnames'
+import { FaAngleRight } from 'react-icons/fa'
+import { periphery } from './data'
+
+interface MyProps {
+  title: string
+  desc: string
+  label: string
+  type?: string
+  id?: string
+}
+
+export const PayoutLHS = ({ title, desc, label, id, type }: MyProps) => {
+  return (
+    <>
+      <h1 className='heading-3-b lg:heading-2-b'>{title}</h1>
+      <p className='paragraph-2'>{desc}</p>
+      <form className='flex flex-col space-y-6'>
+        <label className='font-proxima -mb-4 capitalize paragraph-1-s'>
+          {label}
+        </label>
+        <input
+          type={type}
+          id={id}
+          className={classNames(
+            Styles.input,
+            'placeholder:px-5  placeholder:font-proxima',
+          )}
+          placeholder='Enter your email address'
+        />
```

Comment:
I don't see any `onChange func`. I'm also wondering if it'll be better to have this label and input as a reusable component? Something like `LzInput`